src/js/OTSession.coffee

```diff
@@ -110,7 +110,10 @@ class TBSession
     console.log("JS: Unpublish")
     element = document.getElementById( @publisher.domId )
     if(element)
-      element.parentNode.removeChild(element)
+      element.firstChild && element.removeChild(element.firstChild);
```

Comment:
you can use `this.resetElement` to accomplish what you're adding here

Reply:
Adjusted and used the helper method